charters/IngestSupport/charter.md

```diff
@@ -0,0 +1,100 @@
+# Ingest Support
+
+## Description
+
+The Ingest Support team acts as a liaison between labs who want to contribute data to the Human Cell Atlas (HCA) and the Data Coordination Platform (DCP). This team works with data contributing labs, to understand each lab's data and ensure the experiments are well-represented in the platform and to support the lab writing data to the platform, working with them to resolve any issues in the process.
+
+## Definitions
+
+*Data contributor* - any user who uploads data and metadata into the DCP
+
+*Data wrangler* - a member of the ingest support team who supports to data contributors wanting to upload data into the DCP
+
+*Submission* - the grouping of data and metadata that makes sense to a data contributor. For example, a data contributor might typically upload all the data at once for a publication they have in preparation. This grouping does not necessarily correspond to how data is stored or displayed in the DCP and how it should be displayed to a data consumer
+
+*Data* - raw data files from an experiment such as fastq files for sequencing experiments or tiff stacks for imaging experiments
+
+*Metadata* - descriptive information about what projects, samples and experiments are and how they were collected or conducted
+
+*New data types* - experimental data types which either can't already be ingested by the DCP or have only recently (~6m - 1yr) started to be ingested by the system
```

Comment:
What is an _**experimental** data type_? Is it a data type used in an experiment -or- is the data type itself _experimental_? I became confused when I read about established experimental data types below.

Reply:
That is reasonable. I mean a data type generated by an experiment. I will see if I can improve the phrasing to make this clearer

Reply:
For now I have left the phrasing the same but changed the word to assay

An alternative would be to have

New data types - data types generated by experimental assays that either can't already be ingested by the DCP or have only recently (~6m - 1yr) started to be ingested by the system